public/components/ui_action_context_menu.tsx

```diff
@@ -38,12 +42,20 @@ export const ActionContextMenu = () => {
   return (
     <EuiPopover
       button={
-        <EuiButtonIcon
+        <EuiButtonEmpty
+          color="text"
           aria-label="AI assistant trigger button"
-          size="s"
-          iconType={assistantTriggerIcon}
+          size="xs"
```

Comment:
maybe we should keep it as `s` to align with DQL

![image](https://github.com/user-attachments/assets/a30486f1-7e73-405e-b037-37766fe166e4)

Reply:
having an alignment issue now when changed to a `s` button, this requires a css tweaks in core, I will keep as is now.
![image](https://github.com/user-attachments/assets/3719bbaa-93b5-4952-9702-ae005832fb27)